spec/features/publish/new_course_spec.rb

```diff
@@ -0,0 +1,243 @@
+require "rails_helper"
+
+feature "new course" do
```

Comment:
we could put this in an `e2e` directory under `/features/publish/` similar to how we did it in register

Reply:
done in https://github.com/DFE-Digital/teacher-training-api/pull/2525/commits/77827debc4ed2be38058f60183926ff5264ee34a